src/microbe_stage/MicrobeAI.cs

```diff
@@ -106,175 +106,179 @@ private enum LifeState
 
     public void Think(float delta, Random random, MicrobeAICommonData data)
     {
-        _ = delta;
+        // Disable most AI in a colony
+        if (microbe.Colony?.Master == null)
```

Comment:
You should do an early return if this is not null, so you don't have to indent the entire rest of the AI code.

Reply:
but I still require the calls after the if

Reply:
Then put everything inside the if into a new method, that way the old indentation level can be kept.